utils/src/integ-test/java/uk/gov/moj/sdt/test/utils/IExecuteScriptService.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.moj.sdt.test.utils;
+
+import org.springframework.core.io.Resource;
+
+import java.util.concurrent.Future;
+
+public interface IExecuteScriptService {
+
+    /**
+     * Execution of given script
+     * @param script script resource
+     * @return boolean success/failure
+     */
+    boolean executeScript(Resource script);
+
+    /**
+     * Asynchronous execution of given script
```

Comment:
End the first line with a full stop.  If checkstyle is enabled this will be flagged as "First sentence of JavaDoc missing an ending period." error (SummaryJavadoc rule).